v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/utils/DataflowWorkerMachineTypeValidator.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.spanner.migrations.utils;
+
+public class DataflowWorkerMachineTypeValidator {
+
+  public static void validateMachineSpecs(String workerMachineType, Integer minCPUs) {
+    if (workerMachineType == null || workerMachineType.trim().isEmpty()) {
```

Comment:
[nit ] would it be more idomatic to use [StringUtils.isBlank()](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank(java.lang.CharSequence)) from apache commons for this check and `Preconditions.checkArgument` instead of the explicit throw?